### PR TITLE
waymore: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/waymore.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+> Fetch URLs of a domain from Wayback Machine, Common Crawl, Alien Vault OTX, URLScan, and VirusTotal.
+> Note: Unless specified, output is dumped into `results/` directory where waymore `config.yml` resides (by default in `~/.config/waymore/`).
+> More information: <https://github.com/xnl-h4ck3r/waymore>.
+
+- Search for URLs of a domain (output will typically be in `~/.config/waymore/results/`):
+
+`waymore -i {{example.com}}`
+
+- Limit search results to only include list of URLs for a domain, with outputs sent to a specific file:
```
Comment (gutjuri) [inline pages/common/waymore.md:11]:
```suggestion
- Limit search results to only include a list of URLs for a domain and store outputs to the specified file:
```

File: pages/common/waymore.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`waymore -mode U -oU {{path/to/example.com-urls.txt}} -i {{example.com}}`
+
+- Limit search results to only include content body of URLs of a domain, with outputs sent to a specific directory:
```
Comment (gutjuri) [inline pages/common/waymore.md:15]:
```suggestion
- Only output the content bodies of URLs and store outputs to the specified directory:
```

File: pages/common/waymore.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`waymore -mode R -oR {{path/to/example.com-url-responses}} -i {{example.com}}`
+
+- Limit search to limit results within specific dates:
```
Comment (gutjuri) [inline pages/common/waymore.md:19]:
```suggestion
- Filter the results by timestamp:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions-

File: pages/common/waymore.md (hunk 3 above)
Comment (fazlearefin) [reply pages/common/waymore.md:19]: The word _timestamp_ here does not sound right to me; the switches used are `-to` and `-from`. How about:

```
Filter the results by specifying date ranges:
```

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**